@app/config/src/validate.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Validation functions for environment variables.
+ *
+ * `name` is passed with `value` so that bundlers that replace
+ * process.env.MY_VAR with the value will still provide useful error messages.
```

Comment:
A mini validation framework:
- `required`
- `defaultNumber`
- `minLength`
- `parseInteger`

Other options were using `envalid`, `io-ts`, etc. I think it's simple enough that isn't needed.